docs/src/puppeteer-js.md

```diff
@@ -0,0 +1,168 @@
+---
+id: puppeteer
+title: "Migrating from Puppeteer"
+---
+
+<!-- TOC -->
+
+## Migration Principles
+
+This guide describes migration to [Playwright Library](./library) and [Playwright Test](./intro.md) from Puppeteer. The APIs have similarities, but Playwright offers much more possibilities for web testing and cross-browser automation.
+
+- Most Puppeteer APIs can be used as is
+- The use of [ElementHandle] is discouraged, use [Locator] objects and web-first assertions instead.
+- Playwright is cross-browser
+- You probably don't need explicit wait
+
+## Cheat Sheet
+
+| Puppeteer                                          | Playwright Library                          |
+|----------------------------------------------------|---------------------------------------------|
+| `await puppeteer.launch()`                         | `await playwright.chromium.launch()`        |
```

Comment:
This looks much better, thanks!

Can we list the actions here?
- page.click => page.locator.click
- ...

And I would add a separate list of assertions such as `exepect(locator).toHaveText()`. The reality is that Pptr scripts will be heavy on `page.evaluate` and `page.waitForFunction` for things like getting and waiting for the `.textContent`, `.getAttribute`, etc., etc. I think it would be useful to mention it here since it would make the migrated script so much better (more readable and not flaky).

Reply:
I've listed several actions : `check`, `click`, `focus`, `hover`, `select`, `tap`, `type`

There are now mentions of `$eval` vs web-first assertions in the cheat sheet, test example and the test section.